installer_scripts/install-yb-voyager

```diff
@@ -299,6 +304,20 @@ update_bashrc() {
 	done
 }
 
+insert_yb_session_vars_file() {
+	sudo touch /etc/ybSessionVariables.sql
+	sudo sh -c 'printf "SET client_encoding TO '\''UTF8'\'';\nSET session_replication_role TO replica;" > /etc/ybSessionVariables.sql'
+}
+
+insert_ora2pg_conf() {
+conf_file_name="/etc/yb-voyager-base-ora2pg.conf"
+sudo touch $conf_file_name
+if ! grep -qxF "####################  Ora2Pg Configuration file   #####################" $conf_file_name; then 
+	sudo wget -q https://github.com/yugabyte/yb-voyager/raw/main/yb-voyager/src/srcdb/data/sample-ora2pg.conf
+	sudo mv sample-ora2pg.conf $conf_file_name
+fi
+}
```

Comment:
Properly indent the code.
The file path should be `/etc/yb-voyager/base-ora2pg.conf`.
Right at the beginning of this function, you should check whether the already exists. If yes, then return immediately without doing anything.
Then you won't need the `grep -qxF ...` check.

You should also consider using `-O filename` option of wget so that you won't need the additional step to `mv` the file to the final location.